me.jtheoof.swappy.metainfo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop">
+    <id>me.jtheoof.swappy</id>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>MIT</project_license>
+    <name>swappy</name>
+    <summary>Wayland native snapshot editing tool, inspired by Snappy on macOS</summary>
+    <icon type="remote" width="128" height="128">https://github.com/jtheoof/swappy/raw/master/res/icons/hicolor/scalable/apps/swappy.svg</icon>
+    <description>
+        <p>A Wayland native snapshot and image editing tool, inspired by Snappy on macOS. Works great with grim, slurp and sway. But can easily work with other screen copy tools that can output a final image to stdout.</p>
+    </description>
+    <categories>
+        <category>Utility</category>
+        <category>Graphics</category>
+    </categories>
+    <keywords>
+        <keyword>wayland</keyword>
+        <keyword>snapshot</keyword>
+        <keyword>annotation</keyword>
+        <keyword>editing</keyword>
+    </keywords>
+    <content_rating type="oars-1.1" />
+    <url type="homepage">https://github.com/jtheoof/swappy</url>
+    <url type="help">https://github.com/jtheoof/swappy</url>
+    <url type="bugtracker">https://github.com/jtheoof/swappy/issues</url>
+    <url type="translate">https://github.com/jtheoof/swappy?tab=readme-ov-file#i18n</url>
+    <url type="vcs-browser">https://github.com/jtheoof/swappy</url>
+    <url type="contribute">https://github.com/jtheoof/swappy?tab=readme-ov-file#contributing</url>
+    <provides>
+        <binary>swappy</binary>
+        <mediatype>image/png</mediatype>
+        <mediatype>image/jpeg</mediatype>
+    </provides>
+    <developer id="io.github.jtheoof">
+        <name>Jeremy Attali</name>
+        <url>https://github.com/jtheoof</url>
+    </developer>
+    <screenshots>
+        <screenshot type="default">
+            <image type="source" width="1347" height="672">https://raw.githubusercontent.com/jtheoof/swappy/master/docs/images/screenshot-1.0.0.png</image>
```

Comment:
I can't wait to see this fail when the master branch changes and the 1.0.0 is removed.

This URL should point something more stable like a commit.